run_calculate_loanbook_coverage.R

```diff
@@ -59,42 +68,27 @@ matched_prioritized <- readr::read_csv(
 )
 
 # optional: apply sector split----
-if (apply_sector_split) {
-  companies_sector_split <- readr::read_csv(
-    file.path(input_path_matched, "companies_sector_split.csv"),
-    col_types = col_types_companies_sector_split,
-    col_select = dplyr::all_of(col_select_companies_sector_split)
-  )
-
-  abcd_id <- abcd %>%
-    dplyr::distinct(.data$company_id, .data$name_company)
+if (apply_sector_split & sector_split_type_select %in% c("equal_weights", "worst_case")) {
```

Comment:
NIT: Similar to above. Second assertion is superfluous

Reply:
simplified